### PR TITLE
Virology Symptom Update : Now Featuring EVEN MORE PWRGAME

### PR DESCRIPTION
**Pull request by Psody-Mordheim**
## About The Pull Request
More or less a copy of https://github.com/tgstation/tgstation/commit/f2fad8fcca5876e1a1e7c2aaa7dedd6fa7eaa653#diff-0f059a6536c1df3e6b927d9574898796 on /tg/ Props to bgobandit obviously as the original PR maker.

However this PR also reenables the genetic.dm symptom which for some reason was forgotten in the dme? I can't seem to find any reason besides a single commit by gommie to exclude the symptom for some reason?

https://github.com/Citadel-Station-13/Citadel-Station-13/commit/367789fda4db6949ef908151600e1e8088ceec4e#diff-3835aabe9f262fdddf9d5322ac232ba7

As far as I can tell this was excluded due to it being an error because the genetics dm wasn't included in the dme file and apparently forgotten about for forever.

> But Von Carstein, you fucking powergamer random Mutations are cancer why are you letting virology do this?

Because virology already has half a dozen better ways to kill you then random mutations my guy/girl, I think re-enabling the symptom is overall a net gain and improves the compacity for !FUN!

> Virology is already too powerful though?

I dunno how letting virologists choose random mutations instead of the usual spontaneous combustion, choking or blindness is that much /more/ power-creep, again they already have way better symptoms to fuck your day up, then again I'm somewhat bias towards virology so take my opinion with a grain of salt.

> You realize every sentient virus is gonna take the random mutation symptom to fuck with people, right?

¯\_(ツ)_/¯ if it gets to annoying we can remove that specific line I guess, but then again you know you could just get good and kill the virus.

## Why It's Good For The Game

Ultimately both Vitiligo and Revitiligo are terrible symptoms, I mean really they are just boring space wasters that nobody ever uses.

Whereas Disfiguration and Random Mutation Symptoms are actually ya know useful, wanna turn the entire station into unknowns so beepsky arrests everyone? Well now you can!

And Polyvitiligo makes pretty colors so thats pretty much obviously a benefit, honk.

## Changelog
:cl:
add: Disfiguration Symptom.
add: Deoxyribonucleic Acid Saboteur Symptom.
add: Polyvitiligo Symptom.
del: Revitiligo Symptom.
del: Vitiligo Symptom.
/:cl:

### CODE REVIEW

**Comment by deathride58** [comment]
lore plasma 2: electric boogaloo

**Comment by Psody-Mordheim** [comment]
> lore plasma 2: electric boogaloo

Plasma is good for you, trust me I'm a medical professional, now drink this bottle labeled instant death

File: code/datums/diseases/advance/symptoms/skin.dm [hunk 1]
```diff
@@ -72,17 +26,16 @@ BONUS
...
-		switch(A.stage)
-			if(5)
-				H.skin_tone = "african2"
-				H.update_body(0)
-			else
-				H.visible_message("<span class='warning'>[H] looks a bit dark...</span>", "<span class='notice'>Your skin suddenly appears darker...</span>")
+	switch(A.stage)
+		if(5)
+			var/static/list/banned_reagents = list(/datum/reagent/colorful_reagent/crayonpowder/invisible, /datum/reagent/colorful_reagent/crayonpowder/white)
+			var/color = pick(subtypesof(/datum/reagent/colorful_reagent/crayonpowder) - banned_reagents)
```
Comment (Trilbyspaceclone) [inline code/datums/diseases/advance/symptoms/skin.dm:36]: This one dosnt work as crayons now dont colour people

**Comment by silicons** [comment]
gamer carstein is a coder uwu

**Review by Ghommie — APPROVED** [review]

**Comment by Ghommie** [comment]
If you want, you could also port \tg\ #46347.

**Comment by Xantholne** [comment]
Finally, some good fucking food.

**Pull request merged**